RelhaxModpack/RelhaxModpack/Utils/Translations.cs

```diff
@@ -1938,7 +1936,7 @@ public static void LoadTranslations()
             Polish.Add("expandNodesDefault2", "Rozwiń wszystko"); //? Is this even used? @Nullmaruzero
             French.Add("expandNodesDefault2", "Développer tout");
             Spanish.Add("expandNodesDefault2", "Expandir todo");
-            Russian.Add("expandNodesDefault2", "Раскрыть все");
+            Russian.Add("expandNodesDefault2", "Раскрыть все"); // It seems that's not used. - DrWeb7_1 replying to @Nullmaruzero
```

Comment:
@SigmaTel71 @Nullmaruzero I probably used it at some point in Relhax V1 and copied it over to Relhax V2. If it's not used, don't worry about it.